index.html

```diff
@@ -1,215 +1,284 @@
+                    <div class="langImg">
                         <img src="images//html-Img.png" alt="html" />
-                      </div>
-                      <h4>HTML</h4>
-                    </li>
-                    <li class="lang">
-                      <div class="langImg">
+                    </div>
+                        <h4>HTML</h4>
+                </li>
+                <li class="lang">
+                    <div class="langImg">
                         <img src="images/css-Img.png" alt="css" />
-                      </div>
-                      <h4>CSS</h4>
-                    </li>
-                </ul>
-                    <li class="skills">
-                        <h3 class="langTitle">Frameworks</h3>
-                        <div class="arrow">
+                    </div>
+                        <h4>CSS</h4>
+                </li>
+            </ul>
+                <li class="skills">
+                    <h3 class="langTitle">Frameworks</h3>
+                    <div class="arrow">
                         <img src="images/Arrow2.png" alt="arrow" />
-                        </div>
-                    </li>
+                    </div>
+                </li>
                     <hr />
-                    <li class="skills">
-                        <h3 class="langTitle">Skills</h3>
-                        <div class="arrow">
+                <li class="skills">
+                    <h3 class="langTitle">Skills</h3>
+                    <div class="arrow">
                         <img src="images/Arrow2.png" alt="arrow" />
-                        </div>
-                    </li>
-                    <hr />     
+                    </div>
+                </li>
+                    <hr/>
             </ul>
         </div>
+
+      <!-- Contact Form -->
+        <section class="contact-Container" >
+            <div class="contact-Form">
+                <div class="contact-Me">
+                    <h1>Contact me</h1>
+                    <p>
+                        If you have an application you are interested in developing, a feature
+                        that you need built or a project that needs coding. I'd love to help
+                        with it
+                    </p>
+                </div>
+                <form action="https://formspree.io/f/mdojyaar" method="POST" class="input-Form">
+                    <label for="name">
+                        <input type="text" id="name" name="name" minlength="3" maxlength="30" placeholder="Full name" required/>
+                    </label>
+                    <label for="email">
+                        <input type="email" id="email" name="email" placeholder="Enter the email" required/>
+                    </label>
+                    <label for="message">
+                        <textarea id="message" cols="40" rows="10" required maxlength="500" minlength="10" name="message"placeholder="Write your message here"></textarea>
```

Comment:
- [ ] You did a great job fixing the issues the previous code reviewer mentioned. But the `textarea` of your contact form is still larger. I would recommend you remove the `cols` and `rows` attributes and use `width` and `height` to style it on your CSS file. Kindly fix it so the `textarea` is aligned with the rest of the inputs.